config/crd/bases/stunner.l7mp.io_gatewayconfigs.yaml

```diff
@@ -69,6 +69,10 @@ spec:
                   STUNner relay connections.
                 format: int32
                 type: integer
+              metricsEndpoint:
```

Comment:
CRD changes so unfortunately there are some further steps needed:
* run `make manifests` and `make generate`, and make sure the manifests are indeed updated (should appear here in the changed files list, or did I miss smthg?) worth issuing a `make deploy` into a working cluster and give it a try too  
* Helm chart must be updated: clone https://github.com/l7mp/stunner-helm, copy the new gatewayconfig CRD there: once we tag the release in the operator side, the helm chart gets automatically updated and people get our new CRD
* Update the docs here: https://github.com/l7mp/stunner/blob/main/doc/GATEWAY.md#gatewayconfig

Reply:
Oh, and don't forget to use the new git message format, see the discord

Reply:
Thanks!

Please check these related PRs:
Helm update: https://github.com/l7mp/stunner-helm/pull/3
Stunner doc update: https://github.com/l7mp/stunner/pull/41